config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -84,7 +84,7 @@ presubmits:
         - name: FEATURE_GATES
           value: '{"VolumeAttributesClass":true, "HonorPVReclaimPolicy": true, "CSIVolumeHealth": true}'
         - name: RUNTIME_CONFIG
-          value: '{"api/ga":"true", "storage.k8s.io/v1alpha1":"true"}'
+          value: '{"api/ga":"true", "storage.k8s.io/v1alpha1":"true", "storage.k8s.io/v1beta1":"true"}'
```

Comment:
this enables the API objects, but not the feature gates, so it is not adding much more coverage

Reply:
yes, this PR just enable the beta api because we want to promote the feature-gate and use beta api.